masterlist.yaml

```diff
@@ -9021,6 +9021,11 @@ plugins:
           - 'Open Cities Skyrim'
           - '[AI Overhaul - Open Cites Compatibility](https://www.nexusmods.com/skyrimspecialedition/mods/38031)'
         condition: 'active("Open Cities Skyrim.esp") and not active("AI Overhaul - Open Cities.esp")'
+      - <<: *patch3rdParty
+        subs:
+          - 'Immersive College of Winterhold'
+          - '[Immersive College of Winterhold - AI Overhaul Patch](https://www.nexusmods.com/skyrimspecialedition/mods/41694)'
+        condition: 'active("CollegeOfWinterholdImmersive.esp") and not active("AI Overhaul - CWI Patch.esp")'
```

Comment:
I would put this and the change to `CollegeOfWinterholdImmersive.esp`cleaning info in a separate pull request.
```suggestion
```